core/string/ustring.cpp

```diff
@@ -4618,6 +4639,13 @@ String String::json_escape() const {
 	return escaped;
 }
 
+String String::bbcode_escape() const {
+	String escaped = *this;
+	escaped = escaped.replace("[", "[lb]");
+
+	return escaped;
+}
```

Comment:
The doc mentions `[rb]`, but it's not here.

Reply:
This is an optimization: we don't need to replace the right bracket as the left bracket was already replaced.

That said, we could do it just to be safe. It probably won't be much slower anyway.

cc @bruvzg